CHANGELOG.md

```diff
@@ -2,6 +2,131 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 55 end 2017-02-27
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+55+Ending+Feb+27%2C+2017%22+label%3Aenhancement)
+
+- Automate
+  - Ansible
+    - Add Run Ansible Playbook control action type [(#13943)](https://github.com/ManageIQ/manageiq/pull/13943)
+    - Create service template with Ansible Tower after first creating a new job template [(#13896)](https://github.com/ManageIQ/manageiq/pull/13896)
+    - Added "Ansible Playbook" to the list of catalog item types [(#13936)](https://github.com/ManageIQ/manageiq/pull/13936)
+    - Expose job_template from a service template [(#13895)](https://github.com/ManageIQ/manageiq/pull/13895)
+    - Create catalog item after job templates are created [(#13893)](https://github.com/ManageIQ/manageiq/pull/13893)
+    - Create temporary inventory when execute a playbook [(#14008)](https://github.com/ManageIQ/manageiq/pull/14008)
+    - Run a control action to order Ansible Playbook Service [(#13874)](https://github.com/ManageIQ/manageiq/pull/13874)
+  - Orchestration: Use task queue for update stack operation [(#13897)](https://github.com/ManageIQ/manageiq/pull/13897)
+  - Services
+    - Add service object to deliver to automate. [(#13956)](https://github.com/ManageIQ/manageiq/pull/13956)
+    - Create a provision request for a service template [(#13972)](https://github.com/ManageIQ/manageiq/pull/13972)
+    - ServiceTemplate update_catalog_item [(#13811)](https://github.com/ManageIQ/manageiq/pull/13811)
+  - See also [Manageiq/manageiq-content](https://github.com/ManageIQ/manageiq-content)
+
+- Platform
+  - Chargeback
+    - Chargeback without C & U [(#13884)](https://github.com/ManageIQ/manageiq/pull/13884)
+    - Containers: Enterprise rate parent for chargeback [(#14079)](https://github.com/ManageIQ/manageiq/pull/14079)
+  - Ansible: Properly monitor the embedded ansible service [(#13978)](https://github.com/ManageIQ/manageiq/pull/13978)
+  - See also [Manageiq/manageiq-appliance](https://github.com/ManageIQ/manageiq-appliance)
+
```

Comment:
There's also this addition on the platform side -
[Add a #backlog method to PglogicalSubscription objects](https://github.com/ManageIQ/manageiq/pull/14010)